tensorflow-core/tensorflow-core-api/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Script to build native TensorFlow libraries
+set -eu
+
+# Allows us to use ccache with Bazel
+export BAZEL_USE_CPP_ONLY_TOOLCHAIN=1
+export BAZEL_VC="C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/"
+export PYTHON_BIN_PATH=$(which python.exe)
+if [[ ! -f $PYTHON_BIN_PATH ]]; then
+    export PYTHON_BIN_PATH=$(which python3)
+fi
+
+# Build TensorFlow itself
+bazel build --python_path="$PYTHON_BIN_PATH" --config=mkl --output_filter=DONT_MATCH_ANYTHING --verbose_failures @org_tensorflow//tensorflow:tensorflow @org_tensorflow//tensorflow/java:tensorflow
```

Comment:
Can we build only targets that we actually need for the Java part? We don't have to build the whole client, afaik we just need the operation wrappers, which are provided by this target `//tensorflow/java:java_op_gen_sources`, in replacement to `//tensorflow/java:tensorflow`.

Reply:
No, we can't, unfortunately, that is, not without patching TensorFlow, those targets are not public. Though, I've had to add a patch for MKL, since the fix isn't in 2.0.0. We can add more patches if we want to go that way.

Reply:
Yeah, I think we can have @sjamesr commit this very quickly in the main repository, I'll check with him but not super important neither, we don't have to wait for this.